src/page/RestaurantPage.js

```diff
@@ -36,7 +37,7 @@ class RestaurantPage extends Component {
     const { cart } = this.state
     cart.addMenuItem(menuItem)
 
-    this.cartFooter.animate()
+    this.cartFooter.getWrappedInstance().animate()
```

Comment:
Note that this is because `CartFooter` is wrapped in a HOC using `translate`. See c6fec3d.

This is a simple solution, but I don't know if it's the best one, since it relies on `RestaurantPage` understanding that `CartFooter` is wrapped. Ultimately I think this pattern of calling a method on a child component in the parent is breaking encapsulation. Ideally the `animate` method would be called by `CartFooter` triggered by a change to props or something.

Anyway, I'm happy to leave it like this, I just wanted to draw your attention to it.

Also, if you know of anywhere else in the codebase where this pattern is used, please let me know, and I will double check this whether this HOC-wrapping bug is present. Thanks. 😃